src/test/java/htsjdk/samtools/util/IntervalListTest.java

```diff
@@ -463,16 +469,16 @@ public void testOverlapsSingletonAsListIntervalList(final IntervalList fromLists
                 CollectionUtil.makeCollection(list.iterator()));
     }
 
-    @Test(expectedExceptions = SAMException.class)
+    @Test
     public void testOverlapsEmptyFirstList() {
-        IntervalList.overlaps(Collections.emptyList(), Collections.singletonList(list1));
+        Assert.assertEquals(
+                IntervalList.overlaps(empty, list1), empty);
```

Comment:
Why not just add these as additional test cases in the existing data providers ?

Reply:
nice suggestions. done.